app/views/pages/about.slim

```diff
@@ -80,37 +80,19 @@ section#features.phr.pv4.pv6-m.bg-tiber.white-90
               a.f6 href="https://blog.coinfi.com/how-you-can-make-50-profit-in-12-hours-trading-neo/" target="_blank"
                 strong Real Example Of Volume Detection: 50% Gains In 12 Hours Trading NEO
 
-section#team.phr.bg-white.tiber
-  .container
-    .row.pv5.items-center
-      .col-xs-12.col-lg-8.col-lg-offset-2
-        h1 The Team
-      .col-xs-12.col-lg-2.col-lg-offset-2
-        .pa4.tc
-          img class="br-100 dib" src="https://res.cloudinary.com/coinfi/image/upload/v1522671951/profile-photos/tim-tam.jpg" width="256" alt="Timothy Tam"
-      .col-xs-12.col-lg-6
-        h3
-          | Timothy Tam
-          small.ml3.fw1 Co-Founder
-        p 10+ years in equities trading and hedge funds. Started his career in Goldman Sachs Asia, before working as a senior trader in two Asia hedge funds each with $1+ billion assets under management. In his last role he was cofounder of an Asia Long Short Hedge Fund that specialized in event trading.
-        = render partial: 'linkedin_icon', locals: {url: 'https://www.linkedin.com/in/ttam-0190212/'}
-
-      .col-xs-12.col-lg-2.col-lg-offset-2
-        .pa4.tc
-          img class="br-100 dib" src="https://res.cloudinary.com/coinfi/image/upload/v1522671945/profile-photos/han-chang.jpg" width="256" alt="Han Chang"
-      .col-xs-12.col-lg-6
-        h3
-          | Han Chang
-          small.ml3.fw1 Co-Founder
-        p Master's in Computer Science from Boston University. Early Bitcoin investor, founded a Chinese Bitcoin exchange; previously at MIT Lincoln Labs, OkCupidLabs, General Assembly, and Lime Brokerage. 8+ years of professional experience developing web applications serving 10M+ users as well as writing black box equity trading algorithms.
-        = render partial: 'linkedin_icon', locals: {url: 'https://www.linkedin.com/in/hanchang0/'}
-
-      .col-xs-12.col-lg-2.col-lg-offset-2
-        .pa4.tc
-          img class="br-100 dib" src="https://res.cloudinary.com/coinfi/image/upload/v1522671949/profile-photos/nate-tsang.jpg" width="256"alt="Nate Tsang"
-      .col-xs-12.col-lg-6
-        h3
-          | Nate Tsang
-          small.ml3.fw1 Co-Founder
-        p Holds a Juris Doctor from UAlberta Law. 7+ years of experience as a growth hacker in media startups as well as developing multiple web properties that have cumulatively grown from zero to 10M+ pageviews. Co-founder of a keyword intelligence platform for digital marketers, acquired in 2016.
-        = render partial: 'linkedin_icon', locals: {url: 'https://www.linkedin.com/in/nate-tsang-3a493a153/'}
+section#team.phr.pv5.pv6-m.bg-white.tiber
+  .container-narrow
+    h1.tc.tl-m The Team
+    - TeamMember.all.each do |m|
```

Comment:
Would prefer to use descriptive names whenever possible please, i.e. `member` or `team_member`